net/hcxdumptool/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2009-2014 OpenWrt.org
+# Copyright (C) 2019 Andreas Nilsen <adde88@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hcxdumptool
+PKG_VERSION:=5.1.7
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/github.com/ZerBea/hcxdumptool/releases/download/v$(PKG_VERSION)
+PKG_HASH:=6ac996a506cb312a5f1c5987f30a4a80c793993908750f69f2df51056f961269
+
+PKG_BUILD_PARALLEL:=1
+
+PKG_USE_MIPS16:=0
```

Comment:
Is there a reason for this?

Reply:
There was a reason for it when I added it. But it seems to build fine now without it.
So I've omitted it.